<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QQMainWindow</class>
 <widget class="QMainWindow" name="QQMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QQMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelBackground">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>441</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/login/image/login.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="picturelabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/login/image/avatar.svg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
font: 12pt &quot;ADMUI3Lg&quot;;
color: rgb(170, 85, 255);
font: 9pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>注册账号</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>173</y>
      <width>206</width>
      <height>103</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="userNamelineEdit">
       <property name="inputMask">
        <string/>
       </property>
       <property name="placeholderText">
        <string>账号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pwdlineEdi">
       <property name="placeholderText">
        <string>密码</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="loginradioButton">
         <property name="text">
          <string>自动登陆</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>记住密码</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff55ff;&quot;&gt;找回密码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="loginBtn">
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;微软雅黑&quot;;
background-color: rgb(0, 85, 255);</string>
       </property>
       <property name="text">
        <string>登陆</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
